regcomp.h

```diff
@@ -327,12 +327,12 @@ struct regnode_ssc {
 */
 #if SHORTSIZE > 2
 #  ifndef REG_INFTY
-#    define REG_INFTY  nBIT_UMAX(16)
+#    define REG_INFTY  nBIT_IMAX(32)
 #  endif
 #endif
 
 #ifndef REG_INFTY
-#  define REG_INFTY U16_MAX
+#  define REG_INFTY I32_MAX
```

Comment:
If there are I didn't notice any beyond the tests making assumptions they can create strings that are REG_INFTY long. Ill have a look in the docs. Can you find the commit where you changed it last?

Reply:
FWIW, i found the patch where you did this. And it changes about as much as mine does.  See: 4288c5b93b3823465d8a11e569547afd41b5d1b8

I notice that there has been a lot of work related to REG_INFNTY done since then. I suspect my patch benefits from much of that. Anyway, i will push a fix to the comment. I couldnt find anything else but that.

Reply:
This patch was pushed as https://github.com/Perl/perl5/pull/20706